src/Core/Silk.NET.Core/Native/NativeApiContainer.cs

```diff
@@ -46,10 +36,9 @@ public void Dispose()
             CurrentVTable.Dispose();
         }
 
-        protected virtual int CoreGetSlotCount() => 0;
         protected virtual int CoreGcSlotCount() => 0;
-        protected virtual IVTable CreateVTable() => new ConcurrentDictionaryVTable();
-        protected IVTable SwapVTable() => Interlocked.Exchange(ref _vTable, CreateVTable());
+        protected abstract IVTable CreateVTable();
```

Comment:
Can we make this be virtual but throw instead? I don't want this to be visible to the user and this will likely result in a "abstract method not implemented" warning by most static analysis tools such as IntelliSense and ReSharper.

```suggestion
        protected virtual IVTable CreateVTable()
            => throw new NotImplementedException("Must be implemented by SilkTouch");
```